SQL-Hybrid-Cloud-Toolkit/content/readme.md

```diff
@@ -0,0 +1,35 @@
+# Welcome to the Azure SQL Hybrid Cloud Toolkit!
+## Chapters
+* [Prerequisites and Initial Setup](prereqs.ipynb) - Notebook installation of required modules.
+
+* [Assessments](Assessments/readme.md) - Notebooks that contain examples to determine whether a given database or SQL Server instance is ready to migrate by utilizing SQL Assessments. SQL instances are scanned based on a "best practices" set of rules. 
+
+* [Networking](networking/readme.md) - Setup secure Point-to-Site (P2S) or Site-to-Site (S2S) network connectivity to Microsoft Azure using a Virtual Private Network (VPN). This notebook serves as a building block for other notebooks as communicating securely between on-premise and Azure is essential for many tasks.
+
+* [Provisioning](provisioning/readme.md) - Creating and communicating with SQL Resources in Microsoft Azure. Includes common tasks such as creating SQL Virtual Machines or SQL Managed Instances in the cloud.
+
+* [Data Portability](data-portability/readme.md) - Install a custom Azure function to facilitate importing and exporting cloud resources. The solution uses parallel tasks in Azure Batch to perform data storage work. Azure Batch is a process that runs large-scale parallel and high-performance computing jobs efficiently in Azure. 
+
+* [High Availability and Disaster Recovery](hadr/readme.md) - Notebooks to leverage Azure SQL for business continuity in a hybrid cloud environment.
+
+* [Offline Migration](offline-migration/readme.md) - Notebooks to perform various migrations.
+
+* [Glossary](glossary.md) - set of defined terms.
+
+* [Appendices](appendices.md) - misc info.
+
+## About
+
+The **Azure SQL Hybrid Cloud Toolkit** is a [Jupyter Book](https://jupyterbook.org/intro.html) extension of [Azure Data Studio](https://docs.microsoft.com/en-us/sql/azure-data-studio/download-azure-data-studio) (ADS) designed to help [Azure SQL Database](https://azure.microsoft.com/en-us/services/sql-database/) and ADS users deploy, migrate and configure for a hybrid cloud environment. The toolkit was designed with and intended to be executed within ADS. This is to ensure the best possible user experience for those without vast knowledge of Azure services while adhering closely to the software _best practices_ standards required by experienced cloud users. 
+
+## Goals and Methodology
+The toolkit better positions a customer with regards to planning, migrating, and thriving in a hybrid cloud environment by:
+
+* Providing SQL'zure users with reliable free software and content that is well-written and executable
```

Comment:
Is this supposed to be SQL Azure users?

Reply:
Yep its supposed to be but this was in the original notebook